setuptools/config.py

```diff
@@ -408,7 +408,7 @@ def parsers(self):
             'classifiers': self._get_parser_compound(parse_file, parse_list),
             'license': parse_file,
             'description': parse_file,
-            'long_description': parse_file,
+            'long_description': self._get_parser_compound(parse_list, lambda l: '\n'.join(map(parse_file, l))),
```

Comment:
In place of the lambda, I would use a new method, `self.parse_multi_file` or similar.

Reply:
sounds fair